package.json

```diff
@@ -20,6 +21,7 @@
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "sass": "^1.49.11",
+    "snake-case": "^3.0.4",
```

Comment:
you're using `lodash`, why do you need this package? `lodash` also supports it: https://github.com/lodash/lodash/blob/master/snakeCase.js

Reply:
Removed in 6b86964c2e1dc46a78855c6db1d2228c0c0e2885